nbconvert/exporters/templateexporter.py

```diff
@@ -17,6 +17,7 @@
 from traitlets import HasTraits, Unicode, List, Dict, default, observe
 from traitlets.utils.importstring import import_item
 from ipython_genutils import py3compat
+from jinja2 import TemplateNotFound, Environment, ChoiceLoader, FileSystemLoader
```

Comment:
I think there's another import below that you can remove now that you've added this.